libraries/botbuilder-core/botbuilder/core/teams/teams_info.py

```diff
@@ -318,10 +318,18 @@ def get_team_id(turn_context: TurnContext):
 
     @staticmethod
     async def _get_connector_client(turn_context: TurnContext) -> ConnectorClient:
-        return await turn_context.adapter.create_connector_client(
-            turn_context.activity.service_url
+        if hasattr(turn_context.adapter, 'create_connector_client'):
```

Comment:
I'm not convinced this is required.  JS appears to take some streaming concerns into mind.  Does this work with just getting the ConnectorClient from TurnState as DotNet does?

Reply:
Thanks. Indeed it does work with just getting the ConnectClient from TurnState. 

Though, I encountered a failure in the[ test_on_teams_members_added_activity](https://github.com/microsoft/botbuilder-python/blob/e14f4f4e6a26b439872187afab6f322758730209/libraries/botbuilder-core/tests/teams/test_teams_activity_handler.py#L573-L602) test case. To fix it, I implemented a mock ConnectorClient using the SimpleAdapter. What do you think?